src/Utils/Game.cpp

```diff
@@ -182,4 +182,19 @@ namespace Util
 	{
 		return IsNewFrame(globals::state->frameCount);
 	}
+
+	RE::BGSTextureSet* GetSeasonalSwap(RE::BGSTextureSet* textureSet)
+	{
+		if (textureSet == nullptr) {
+			return nullptr;
+		}
+
+		if (textureSet->pad12C > 0) {
```

Comment:
Do you know what pad12C is? It may be worth upstreaming.

Reply:
It is an unused uint32. Recent seasons version puts the formid swap there so other plugins can read it. It's not used by the game currently